src/server/response.h

```diff
@@ -47,7 +47,7 @@ class Response {
 
     static std::unique_ptr<Response> build(const InternalServer& server);
     static std::unique_ptr<Response> build_304(const InternalServer& server, const ETag& etag);
-    static std::unique_ptr<Response> build_404(const InternalServer& server, const RequestContext& request, const std::string& bookName);
+    static std::unique_ptr<Response> build_404(const InternalServer& server, const RequestContext& request, const std::string& bookName, const std::string& details);
```

Comment:
You can specify a default value of an empty string for `details`. Then you won't have to update all call sites of `build_404()`.

Reply:
oh yes, of course! I'll do that.